.github/PULL_REQUEST_TEMP.md

```diff
@@ -0,0 +1,59 @@
+## Related issue #
+
+<!--
+Please link the GitHub issue this pull request resolves in the format of `#1234`. If you discussed this change
+with a maintainer, please mention her/him using the `@` syntax (e.g. `@JimBugwadia`).
+
+If this change neither resolves an existing issue nor has sign-off from one of the maintainers, there is a
+chance substantial changes will be requested or that the changes will be rejected.
+
+You can discuss changes with maintainers in the [Kyverno Slack Channel](https://kubernetes.slack.com/).
+-->
+
+## Milestone of this PR
+<!--
+
+Add the milestone label by commenting `/milestone 1.2.3`.
+
+-->
+## What type of PR is this
+
+<!--
+
+> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading white spaces from that line:
+>
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
```

Comment:
Some of these are not applicable for website changes. We can probably categorize as:

```
/kind documentation
/kind sample-policies
/kind js-css-hugo
/kind versioning
```